lib/monitor/stats.go

```diff
@@ -2,120 +2,83 @@ package monitor
 
 import (
 	"fmt"
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/montanaflynn/stats"
 	"io"
 	"math"
 	"regexp"
 	"sort"
 	"strconv"
 	"strings"
-
-	"github.com/dedis/cothority/lib/dbg"
-	"github.com/montanaflynn/stats"
 )
 
 // Stats contains all structures that are related to the computations of stats
-// such as Values (compute the mean/min/max/...), Measurements ( aggregation of
-// Values), Stats (collection of measurements) and DataFilter which is used to
+// such as Value (compute the mean/min/max/...), Measurements ( aggregation of
+// Value), Stats (collection of measurements) and DataFilter which is used to
 // apply some filtering before any statistics is done.
 
 // Stats holds the different measurements done
 type Stats struct {
-	// How many hosts do we have
-	Hosts int
-	// How many servers do we have
-	Servers int
-	// How many hosts are ready
-	Ready int
-
```

Comment:
Still curiosity: how do you store those values so they find their way in the .csv-files?